service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java

```diff
@@ -156,7 +148,8 @@ private String buildLotNumberPertinentInformation(Immunization immunization) {
 
     private String buildExpirationDatePertinentInformation(Immunization immunization) {
         Optional<Date> expirationDate = Optional.ofNullable(immunization.getExpirationDate());
-        return expirationDate.map(value -> EXPIRATION + formatShortDate(value)).orElse(StringUtils.EMPTY);
+        return expirationDate.map(date -> EXPIRATION + DateFormatUtil.formatDate(date))
+            .orElse(StringUtils.EMPTY);
```

Comment:
static import EMPTY?

Reply:
It is better to have a name of the class in front for better readability. Simple "EMPTY" does not say much.